assignments/HackYourTemperature/server.js

```diff
@@ -0,0 +1,19 @@
+import express from 'express';
+
+const app = express();
+const port = 3000;
+
+app.use(express.json());
+
+app.get('/', (req, res) => {
+  res.send('Hello from backend to frontend');
+});
+
+app.post('/weather', (req, res) => {
+  const { cityName } = req.body;
+  res.send(`you submitted: ${cityName}`);
```

Comment:
This works but it might not be easy to work with later

Keep in mind that later you will receive a city and will need to return the weather of the city in a json format.

What you could here is either return the cityName directly without the full sentence or return a json object, here is both options as a sugegstions:

Return the cityName itself
```suggestion
  res.send(cityName);
```

Return a json object
```suggestion
  res.send({ cityName });
```